src/components/campaign-types/layout/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import Container from '@mui/material/Container'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+
+interface CopyrightProps {
+  description: string
+}
+
+function Copyright(props: CopyrightProps) {
+  return (
+    <Typography
+      variant="body2"
+      color="text.secondary"
+      align="center"
+      style={{ marginBottom: '0.5%', color: 'white' }}>
+      <Link color="inherit" style={{ fontSize: '13px', textDecoration: 'none' }}>
+        © {new Date().getFullYear()} Pordkrepi.bg Campaign types module {props.description}
```

Comment:
This footer component seems to be duplicated and exists more than once. It's better to have them in a single place and reuse it whenever appropriate.